### PR TITLE
Add English (U.K.) language variant

### PR DESCRIPTION
**Pull request by Coopydood**
Hey, love the project, and wanted to do my part. I'm from the UK, and we have some slight language differences from standard English (US). 

I've added a Localizable file for English (U.K.) / `english_gb`. It covers the few strings that differ from the US variant, namely:

Color > Colour
Utilization > Utilisation
Gray > Grey
Center > Centre

image

Thanks for your work, it's appreciated.

Take care

### CODE REVIEW

**Comment by Coopydood** [comment]
What needs to be changed, @exelban? Not sure why the check is failing :/

File: Stats/Supporting Files/en_gb.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,451 @@
+//
+//  Localizable.strings
+//  Stats
+//
+//  Translated by Coopydood
+//
```
Comment (exelban) [inline Stats/Supporting Files/en_gb.lproj/Localizable.strings:6]: you removed these lines from the header:
//  Using Swift 5.0.
//  Running on macOS 10.15.

and now the line number is not correct

**Comment by exelban** [comment]
Please feel free to leave the author's information on line 5.

**Comment by Coopydood** [comment]
Ready to go when you are :)

**Pull request merged**